api/pom.xml

```diff
@@ -11,4 +12,41 @@
 	<packaging>jar</packaging>
 	<name>International Patient Summary API</name>
 	<description>API project for International Patient Summary</description>
-</project>
+	<dependencies>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-junit-jupiter</artifactId>
+			<version>3.12.4</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>4.0.0</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-inline</artifactId>
+			<version>3.9.0</version>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-module-junit4</artifactId>
+			<version>2.0.9</version>
+		</dependency>
+		<dependency>
+			<groupId>org.powermock</groupId>
+			<artifactId>powermock-api-mockito2</artifactId>
+			<version>2.0.9</version>
```

Comment:
These should probably have `<scope>test</scope>` as well. Usually, the best practice pattern is to define the specific versions and scopes in the `dependencyManagement` section of the main POM then here you only need:

```
<dependency>
	<groupId>org.mockito</groupId>
	<artifactId>mockito-inline</artifactId>
</dependency>
```